managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3473,7 +3473,17 @@ public PositionImpl getNextValidPosition(final PositionImpl position) {
         }
         return next;
     }
-
+    public PositionImpl getValidPositionAfterSkippedEntries(final PositionImpl position, int skippedEntryNum) {
+        PositionImpl skippedPosition = position.getPositionAfterEntries(skippedEntryNum);
+        while (!isValidPosition(skippedPosition)) {
```

Comment:
Is this same as `getNextValidPosition(skippedPostion)`?

Reply:
@Jason918 PTAL,thanks!

The two methods are different.
The getValidPositionAfterSkippedEntries method skips multiple messages at a time, while the getNextValidPosition method skips only one message at a time.

Reply:
Fixed! 
use getValidPositionAfterSkippedEntries for getNextValidPosition: @Jason918 PTAL,thanks!
<img width="1043" alt="image" src="https://user-images.githubusercontent.com/19296967/194686323-6863873a-ffde-49e8-86fe-c306b1f7d4de.png">